systemd/mastodon-es.service

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Elasticsearch for Mastodon
+PartOf=mastodon.service
+Requires=mastodon-net-internal.service
+After=mastodon-net-internal.service
+
+[Service]
+WorkingDirectory=/opt/mastodon
+User=mastodon
+EnvironmentFile=/opt/mastodon/versions.env
+ExecStartPre=+/usr/bin/rm -f elasticsearch/nodes/*/node.lock
+ExecStartPre=+/usr/bin/chown -R 1000:1001 ./elasticsearch
+ExecStart=/usr/bin/docker run --rm --network=mastodon-net-internal \
+                              --health-cmd="curl --silent --fail localhost:9200/_cluster/health || exit 1" \
+                              --volume=/opt/mastodon/elasticsearch:/usr/share/elasticsearch/data \
+                              --env ES_JAVA_OPTS="-Xms512m -Xmx512m -Des.enforce.bootstrap.checks=true" \
+                              --env bootstrap.memory_lock=true \
+                              --env xpack.license.self_generated.type=basic \
+                              --env xpack.security.enabled=false \
+                              --env xpack.watcher.enabled=false \
+                              --env xpack.graph.enabled=false \
+                              --env xpack.ml.enabled=false \
+                              --env ingest.geoip.downloader.enabled=false \
+                              --env cluster.name=es-mastodon \
+                              --env discovery.type=single-node \
+                              --env thread_pool.write.queue_size=1000 \
```

Comment:
Where did you take those flags and values from? Should it be documented too?

Reply:
It's taken from here:https://github.com/mastodon/mastodon/blob/main/docker-compose.yml#L26 and you're absolutely right I should call this out. Will update.

Reply:
Done.